foghorn-edge/templates/edgeml-advanced3-deployment.yaml

```diff
@@ -0,0 +1,64 @@
+{{ if eq .Values.edgeml.type "ADVANCED3"}}
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.